src/common/TrackedOp.h

```diff
@@ -290,6 +295,23 @@ class TrackedOp : public boost::intrusive::list_base_hook<> {
     return events.rbegin()->c_str();
   }
 
+  void set_pg(PG* pg) {
+	  pg_ptr = pg;
+  }
+
+  PG* get_pg() {
+	  return pg_ptr;
+  }
+
+  void set_pglock_can_release_flag(bool flag) {
+      pglock_can_release_flag = flag;
+  }
+
+  bool get_pglock_can_release_flag() {
+      return pglock_can_release_flag;
+  }
```

Comment:
The indentation here is all off from our standards. Please pass it through emacs or vim or something that will follow them. :)